### PR TITLE
[Linux] Wine manager minor update

### PR DESCRIPTION
**Pull request by Nocccer**
- updated to heroic-wine-downloader 1.2.2
- Installation is now abortable
- improved progressbar
image

- names of the version are now correct and unzipped in a folder with the same name

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/screens/WineManager/components/WineItem/index.tsx [hunk 1]
```diff
@@ -110,17 +106,19 @@ const WineItem = ({
...
+        <div className="wineManagerListDate">{date}</div>
+        <div className="wineManagerListSize">{renderStatus()}</div>
         <span className="icons">
           {!isInstalled && !isDownloading && !unZipping && (
             <DownIcon className="downIcon" onClick={() => install()} />
           )}
-          {(isDownloading || unZipping) && <StopIcon onClick={() => null} />}
+          {(isDownloading || unZipping) && (
+            <StopIcon
+              onClick={() => ipcRenderer.send('abortWineInstallation')}
```
Comment (flavioislima) [inline src/screens/WineManager/components/WineItem/index.tsx:119]: it doesn't need to pass the version to abort? what happens if more than one version is being downloaded and we click on abort?

**Review by flavioislima — COMMENTED** [review]
Looks good. Just left one small question.

File: src/screens/WineManager/components/WineItem/index.tsx (hunk 1 above)
Comment (Nocccer) [reply src/screens/WineManager/components/WineItem/index.tsx:119]: Good point. Will fix this later

**Review by flavioislima — DISMISSED** [review]
LGTM 👍🏽

**Review by flavioislima — APPROVED** [review]

**Pull request merged**